hack/tools/containerd-puller/main.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// tiny utility to pull an image to containerd without unpacking it
+// TODO: see if we can ship an option in ctr images pull similar to ctr images import --no-unpack
```

Comment:
[because unpacking needs to create mounts in the snapshotter, and we just want to successfully pull the image into the containerd content store, we care about the image pulling code path, not the image unpacking codepath]